src/main/java/seedu/address/logic/commands/eventcommands/EventFindCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.commands.eventcommands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.model.event.EventNameContainsKeywordsPredicate;
+
+/**
+ * Finds one or more events.
+ */
+public class EventFindCommand extends Command {
+
+    public static final String COMMAND_WORD = "efind";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Finds all events whose names contain any of "
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
+            + "Parameters: n/NAME [n/MORE_KEYWORDS]...\n"
```

Comment:
maybe can be more specific with the `MORE_KEYWORDS`

Reply:
done